src/main/java/com/users/dto/UserDto.java

```diff
@@ -22,7 +24,16 @@ public class UserDto extends AuditorDto {
             regexp = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$",
             message = "Invalid email format"
     )
-    private String email;
+    private String primaryEmail;
+
+    private List<String> secondaryEmails = new ArrayList<>();
+
+    @NotNull(message = "Phone number is mandatory")
+    @Pattern(
+            regexp = "^(\\+4|)?(07[0-9]{8}|02[0-9]{8}|03[0-9]{8})$",
```

Comment:
regexurile astea le poti scaote in niste constante si pune intr o clasa de constante sa le ai intr un singur loc, le poti lasa asa acum, dar pe viitor sa stii

Reply:
am inteles